gitpod-io/gitpod
# Remove service waiter

## Description

Replace `service-waiter` CLI by kubectl wait.

## How to test
- Check the installations works
- Open a workspace

## Release Notes

```release-note
Remove service waiter
```
